sr_robot_launch/scripts/construct_robot_description

```diff
@@ -0,0 +1,78 @@
+#!/usr/bin/env python
+#
+# Copyright (C) 2020 Shadow Robot Company Ltd - All Rights Reserved. Proprietary and Confidential.
+# Unauthorized copying of the content in this file, via any medium is strictly prohibited.
+
+import os
+import rospy
+import rospkg
+from subprocess import check_output
+from sr_ur_arm_calibration_loader.sr_ur_arm_calibration_loader import SrUrLoadCalibration
+
+
+class SrConstructRobotDescription():
+    def __init__(self):
+        self._robot_description_file = rospy.get_param("~robot_description_file")
+        arm_type = rospy.get_param("~arm_type")
+        self._bimanual = False
+        self._xacro_commands = []
+        self._node_name = rospy.get_name()
+
+        if 'bimanual' in self._robot_description_file:
+            self._bimanual = True
+
+        arm1 = ('ra', '192.168.1.1', arm_type)
+        arm2 = ('la', '192.168.2.1', arm_type)
+        if not self._bimanual:
+            self._sr_ur_load_calibration = SrUrLoadCalibration([arm1])
+        else:
+            self._sr_ur_load_calibration = SrUrLoadCalibration([arm1, arm2])
+        self._robot_description_params = self._get_parameters()
+        kinematics_configs = self._get_kinematics_config()
+        if not self._bimanual:
+            self._robot_description_params['kinematics_config'] = kinematics_configs[0]
```

Comment:
i would just rename "get_parameters" it  sth like "process robot description" and add it there